src/workers/cdc.ts

```diff
@@ -259,12 +259,17 @@ const onSettingsChange = async (
 
 const onPostChange = async (
   con: DataSource,
-  logger: FastifyLoggerInstance,
+  logger: FastifyBaseLogger,
   data: ChangeMessage<Post>,
 ): Promise<void> => {
   if (data.payload.op === 'c') {
-    await notifyPostAdded(logger, data.payload.after);
+    if (data.payload.after.visible) {
+      await notifyPostVisible(logger, data.payload.after);
+    }
   } else if (data.payload.op === 'u') {
+    if (!data.payload.before.visible && data.payload.after.visible) {
+      await notifyPostVisible(logger, data.payload.after);
+    }
```

Comment:
Hmm two minds about it, it might become very static then.
And technically they can have multiple actions right?

Community picks = do stuff
Squad = do stuff
Shared post = do stuff

So maybe best to keep this part the same for now?

Reply:
Not sure I understand happy to have a quick chat